scapy/contrib/bgp.py

```diff
@@ -449,7 +476,7 @@ class BGPHeader(Packet):
             0xffffffffffffffffffffffffffffffff,
             0x80
         ),
-        ShortField("len", None),
```

Comment:
This is very not scapy-like. Don't do that. It is commonly understood that `len=None` -> auto-computed.
You can always set this to 0 before calling fuzz